jquery.nested_attributes.coffee

```diff
@@ -161,7 +161,7 @@ class NestedAttributes
       @bindDestroy($record) if @options.$clone or !@options.deepClone
 
       # Empty out the values of text inputs and selects
-      $record.find(':text, textarea, select').val('')
+      $record.find('input:not([type=submit]), textarea, select').val('')
```

Comment:
Thanks @lulalala ! Question: does this liberal selector `input:not([type=submit])` have any affect on the `:checkbox, :radio` on line 167? Have you tested with checkboxes/radios present in the form?